src/components/common/TmSessionExtension.vue

```diff
@@ -29,6 +29,13 @@
           :button-text="`Use Account`"
         />
       </div>
+
+      <div v-else class="session-main">
+        <p class="extension-message">
+          Looks like you don't have any accounts yet. Head over to
+          <router-link to="selectnetwork">create one</router-link>.
```

Comment:
that leads the user to create an account in the browser right?

Reply:
That will only redirect to the create an account page. Now one this page we have a warning about creating accounts in browser with the links to apps, extension and ledger. 
This one 
![](https://user-images.githubusercontent.com/9531274/73543996-f74cb900-4459-11ea-866e-a102bcbea5c9.png)
But with insecure mode the this link will lead to an account creation step, like it should

Reply:
Shouldn't from this the account creation happen in the extension and not locally even for insecure mode?

Reply:
It's a proposal or a suggestion? Anyway, i kept all logic of creation, just added a link there.

Reply:
suggestion. right now this is wrong. when the user (in insecure mode) clicks on "sign in with extension" > "create account" they will be directed to create an account locally instead of in the extension.